### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by balqis277**
Ready

### CODE REVIEW

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

**Comment by ngocvt-0484** [comment]
ready

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
```
Comment (ngocvt-0484) [inline config/database.yml:7]: this file will not be up to GIt, please
```
git rm --cached config/database.yml
```
and in .gitignore add
```
config/database.yml
```

**Comment by balqis277** [comment]
Ready

**Comment by ngocvt-0484** [comment]
commented

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

**Comment by balqis277** [comment]
Ready

File: config/database.yml (hunk 1 above)
Comment (balqis277) [reply config/database.yml]: I already tried this coding. But, when i comment 'ready' , the pull request did not appear in the chatwork.

**Comment by balqis277** [comment]
Ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**